README.md

```diff
@@ -1,24 +1,62 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Hello React Rails
+This is a project created with a react frontend and a rails backend which displays "Hello world" in the following languages: English, French, Spanish, Japanesse, Chinese and Italian.
 
-Things you may want to cover:
 
-* Ruby version
 
-* System dependencies
+## Built With
 
-* Configuration
+  - Ruby/Rails
+  - PostgreSQL
+  - React gem
 
-* Database creation
 
-* Database initialization
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+## Getting Started
 
-* Deployment instructions
+Follow the steps bellow to get a local copy up and running.
 
-* ...
+- Run `git clone https://github.com/Kingstalux/Hello-React-Rails.git` from your terminal to clone the repo from GitHub.
+
+- Run `cd hello-react-rails` to move to the project folder.
+
+-  Run `bundle install` and `npm install` from the command line to install dependencies.
+
```

Comment:
[OPTIONAL] I recommend adding to the Getting started section the following lines after bundle install:
```
Run rails webpacker:install   
Run rails webpacker:install:react 
```
I wasn't able to run your app until I installed `webpacker` and `webpacker:react`.

Reply:
Okay thanks so much, I'll add those now 👍